src/CrossSigningManager.js

```diff
@@ -96,11 +96,8 @@ async function getSecretStorageKey({ keys: keyInfos }, ssssItemName) {
         {
             keyInfo: info,
             checkPrivateKey: async (input) => {
-                if (!info.pubkey) {
-                    return true;
-                }
                 const key = await inputToKey(input);
-                return MatrixClientPeg.get().checkSecretStoragePrivateKey(key, info.pubkey);
+                return await MatrixClientPeg.get().checkSecretStorageKey(key, info);
```

Comment:
I tend to just `return` the result of async functions, as it's [claimed](https://eslint.org/docs/rules/no-return-await) that it creates overhead for some JS engines by adding another layer of promises that aren't really needed... but then while looking that up, I see that it [may produce](https://stackoverflow.com/a/44806230/486978) better stack traces for debugging.

Anyway, we don't have any official style, so fine to leave as-is, since I assume it seemed clearer this way. 😄